tests/aiohttp/test_graphql_transport_ws.py

```diff
@@ -341,7 +341,7 @@ async def test_subscription_syntax_error(aiohttp_client):
         assert response["type"] == ErrorMessage.type
         assert response["id"] == "sub1"
         assert len(response["payload"]) == 1
-        assert response["payload"][0]["path"] is None
+        assert response["payload"][0].get("path") is None
```

Comment:
Why did these get changed to `.get()`? There are two ways this can return `None`, so we should make sure we're checking just one of the means.

Reply:
`path` is not always available anymore: https://github.com/graphql-python/graphql-core/blob/main/src/graphql/error/graphql_error.py#L222-L237